lib/brisket.js

```diff
@@ -37,6 +37,8 @@ var Brisket = {
 
     $: require("./application/jquery"),
 
+    version: require("../package.json").version,
```

Comment:
I agree. But I'm not sure how to not bundle package.json unless we create a separate file with a manually updated version number in it.